### ISSUE TITLE
removeHs should not remove H atoms that determine double bond stereochemistry

### ISSUE THREAD

**Issue by greglandrum** [rdkit/rdkit#1419]
This affects imines:
```
In [14]: Chem.CanonSmiles('C/C(F)=N/[H]')
Out[14]: 'CC(=N)F'
```

One oddity that fixing this would lead to:
```
>>> Chem.CanonSmiles('C/C(F)=N/[H]')
'[H]/N=C(/C)F'
>>> Chem.CanonSmiles('C/C(F)=N[H]')
'CC(=N)F'
```

reported by @coleb here: http://www.mail-archive.com/rdkit-discuss@lists.sourceforge.net/msg06814.html

**Comment by coleb** [comment]
Poke poke, this has never been fixed and @greglandrum said at the UGM to poke about that stuff.

**Comment by greglandrum** [comment]
Fixed by #1810
